src/firefly/js/tables/__tests__/TableUtil-test.js

```diff
@@ -146,10 +146,49 @@ describe('TableUtil: ', () => {
         res = formatValue({format: 'cost $%.2f'}, 123.3432);
         expect(res).toEqual('cost $123.34');
 
-        // @Kartikeya Puri, please add tests for precision column meta here...
+        // @Kartikeya Puri, adding tests for precision column meta here...
         // for precision, column must be numeric.. i.e.  {type: 'float', precision: 'E3'}
         // see TableUtil.js->formatValue function descriptions from details
 
+        //for DMS, islat = true therefore latitude (dec)
+        res = formatValue({type: 'float', precision: 'DMS5'}, "+30.263");
+        expect(res).toEqual('+30d15m46.8s');
+
+        //for HMS, islat = false therefore longitutde (ra)
+        res = formatValue({type: 'float', precision: 'HMS5'}, "+30.263");
+        expect(res).toEqual('2h01m03.12s');
+
+        res = formatValue({type: 'float', precision: 'DMS5'}, "-15.530694"); //latitude/dec
+        expect(res).toEqual('-15d31m50.5s');
+
+        res = formatValue({type: 'float', precision: 'HMS'}, "324.42"); //longitude/ra
+        expect(res).toEqual('21h37m40.80s');
+
+        //400, with DMS is converted to 40 (400 - 360) in dd2sex (CoordUtil.js)
+        res = formatValue({type: 'float', precision: 'DMS5'}, "400"); //latitude/dec
+        expect(res).toEqual('+40d00m00.0s');
```

Comment:
This is good.  Testing out-of-range condition is a good idea.

Reply:
You have lots of tests, but some are repetitive.  
May I suggest...
For HMS, DMS
- HMS; valid, ignored 'n', out-of-range 
- DMS; valid, ignored 'n', out-of-range
Ranges are defined here: edu/caltech/ipac/astro/CoordUtil.java:29
```
       "Latitude is out of range [-90.0, +90.0]";
       "Longitude is out of range [0.0, 360.0)";
```
 For E,G, and F
 - valid, rounding, truncating
 - different numeric types ['int', 'long', 'float', 'double']
 - non-numeric type (should ignore)